packages/flutter/test/material/refresh_indicator_test.dart

```diff
@@ -455,4 +455,71 @@ void main() {
 
     expect(layoutCount, 1);
   });
-}
\ No newline at end of file
+
+  testWidgets('strokeWidth cannot be null in RefreshIndicator', (WidgetTester tester) async {
+    try {
+      await tester.pumpWidget(
```

Comment:
Makes sense. I'll do the required changes.

Reply:
I tried this, but it didn't work as intended. It's was throwing the assertion error and test was failing.
I'm also not able to use 
```dart
expect(tester.takeException(), isAssertionError);
```  
because it won't catch the error in the Constructor. 
For catching the assertion error in Constructor, I'm trying something like this [DropDownButton](https://github.com/flutter/flutter/blob/be3a4b37b3e9ab4e80d45b59bed53708b96d211f/packages/flutter/test/material/dropdown_form_field_test.dart#L567)

Reply:
Ah I see now. Thanks for checking out the different approaches! :)